### ISSUE TITLE
error TS2420: Class 'NgxStorageEvent' incorrectly implements interface 'StorageEvent'.    Property 'initStorageEvent' is missing in type 'NgxStorageEvent' but required in type 'StorageEvent'

### ISSUE THREAD

**Issue by krishna217** [eFaps/ngx-store#1]
After update to Angular v13 and typescript > v4.4 I got the following error during build

```
Error: node_modules/@efaps/ngx-store/lib/utility/storage/storage-event.d.ts:1:22 - error TS2420: Class 'NgxStorageEvent' incorrectly implements interface 'StorageEvent'.

Property 'initStorageEvent' is missing in type 'NgxStorageEvent' but required in type 'StorageEvent'.
export declare class NgxStorageEvent implements StorageEvent {

node_modules/typescript/lib/lib.dom.d.ts:13362:5

13362     initStorageEvent(type: string, bubbles?: boolean, cancelable?: boolean, key?: string | null, oldValue?: string | null, newValue?: string | null, url?: string | URL, storageArea?: Storage | null): void;

'initStorageEvent' is declared here.
```

**Comment by krishna217** [comment]
@jan-moxter Any update ?

**Comment by krishna217** [comment]
Fixed with version 4.2.0. Thanks
